webui/react/src/components/FilterForm/components/FilterField.tsx

```diff
@@ -203,8 +240,10 @@ const FilterField = ({
         />
         <Select
           data-test="operator"
-          options={(isSpecialColumn
-            ? [Operator.Eq, Operator.NotEq] // just Eq and NotEq for Special column
+          options={(columnType !== COLUMN_TYPE.NormalColumnType
+            ? columnType === COLUMN_TYPE.StringMetadataColumnType
+              ? [Operator.Contains, Operator.Eq, Operator.NotEq] // just Contain, Eq and NotEq for String metadata column
+              : [Operator.Eq, Operator.NotEq] // just Eq and NotEq for Special column
             : AvailableOperators[currentColumn?.type ?? V1ColumnType.UNSPECIFIED]
```

Comment:
consider a mapping here:
```suggestion
          options={{
            [COLUMN_TYPE.StringMetadataColumnType]: [Operator.Contains, Operator.Eq, Operator.NotEq],
            [COLUMN_TYPE.SpecialColumnType]: [Operator.Eq, Operator.NotEq]
            [COLUMN_TYPE.NormalColumnType]: AvailableOperators[currentColumn??.type ?? V1ColumnType.UNSPECIFIED]
          }[columnType]
```

Reply:
fixed